### ISSUE TITLE
Tribe Rotation "rotate" REST endpoint

### ISSUE THREAD

**Issue by u-ways** [U-ways/choppa#121]
Implement the rotate controller endpoint only:

![image.png](https://images.zenhubusercontent.com/5f5e6e2ee5c5a6bec5cc5956/867a91e2-2cf2-4c51-97d4-9ca329345fe4)

**NOTE**: The undo rotation has been moved to another ticket to unblock this tick cause of #158.
